packages/stg-evm-v2/devtools/tasks/constants.ts

```diff
@@ -3,6 +3,8 @@ export const TASK_STG_WIRE_CREDIT_MESSAGING = `stg:wire::credit-messaging`
 export const TASK_STG_WIRE_TOKEN_MESSAGING = `stg:wire::token-messaging`
 export const TASK_STG_WIRE_TOKEN_MESSAGING_INITIALIZE_STORAGE = `stg:wire::token-messaging:initialize-storage`
 
+export const TASK_STG_WIRE_MESSAGING_DELEGATE = `stg:conf::messaging:delegate`
```

Comment:
nit: to be consistent,  i think we should change the name here to

```suggestion
export const TASK_STG_WIRE_MESSAGING_DELEGATE = `stg:wire::messaging:delegate`
```